src/Controls/samples/Controls.Sample/MyApp.cs

```diff
@@ -9,15 +9,17 @@ namespace Maui.Controls.Sample
 {
 	public class MyApp : IApplication
 	{
-		List<IWindow> _windows = new List<IWindow>();
-		public IReadOnlyList<IWindow> Windows => _windows.AsReadOnly();
+		readonly List<IWindow> _windows = new();
 
-		public MyApp(IServiceProvider services, ITextService textService)
+		public MyApp(IServiceProvider services, ITextService textService, IImageSourceServiceConfiguration imageConfig)
 		{
 			Services = services;
 			Debug.WriteLine($"The injected text service had a message: '{textService.GetText()}'");
+			imageConfig.SetImageDirectory("Assets");
```

Comment:
Is this also configurable inside the builders? 

Configuring off the App ctor at first seemed weird but it's probably a good thing to have for people that don't want to use builder

I wonder if it'd be weird to have stuff like this accessible via service locator from the base class inside controls

I just worry about discoverability of things like this where you have to know about the interface to add to your constructor

Reply:
yeah... This needs a re-think because I am not sure where we want this to live. I would like a builder-time configuration - just like a logging level or something. Opened an issue to fix this : https://github.com/dotnet/maui/issues/886